src/quo2/components/wallet/account_card/view.cljs

```diff
@@ -47,7 +47,7 @@
 
 (defn- user-account
   [{:keys [state name balance percentage-value loading? amount customization-color type emoji metrics?
-           theme]}]
+           theme on-press]}]
```

Comment:
btw, I think we need to handle the internal state/styling for when this component is pressed - ideally you can copy how it is being done in the quo2/button component 👍 if it's not already there.

Or else we need to create a follow up issue to address this

Reply:
Created an issue https://github.com/status-im/status-mobile/issues/17081

Reply:
@J-Son89 This is not directly related to this PR, but would [account list card](https://github.com/status-im/status-mobile/blob/develop/src/quo2/components/list_items/account_list_card/view.cljs#L12) need to be updated similarly?

Reply:
I will take a look, but probably if it's a similar issue

Reply:
@erikseppanen yep looks like you're right there. seems we have to watch out for this in the prs :)